docs/code-convention.md

```
@@ -0,0 +1,47 @@
+# Planit 프로젝트 코드 컨벤션 가이드
+
+이 문서는 Planit 프로젝트의 일관된 코드 품질과 효율적인 협업을 위한 컨벤션을 정의합니다.
+
+## 1. 개발 컨벤션 (Development Convention)
+
+### 1.1 PR(Pull Request) 컨벤션
+PR 제목은 PR의 목적을 명확히 드러내야 합니다.
+* **형식:** `[커밋 타입] 작업 내용`
+* **예시:** `[FEAT] 카카오 소셜 로그인 구현`
+* **주요 커밋 타입:**
+    * `FEAT`: 새로운 기능 추가
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)